### PR TITLE
Tuesday

### PR DESCRIPTION
**Pull request by Ridalor**
This is a big one

### CODE REVIEW

File: src/templates/orderIndex.html [hunk 1]
```diff
@@ -82,16 +82,18 @@ <h3>Delivery method</h3>
                                     <option value="1">By foot</option>
                                     <option value="2" selected>By car</option>
                                     <option value="3">By transit</option>
-                                    <option value="4">By drone</option>
```
Comment (durdal) [inline src/templates/orderIndex.html]: Aww, no drone :(

Comment (TOsaland) [reply src/templates/orderIndex.html]: :((

File: src/api.py [hunk 2]
```diff
@@ -43,58 +42,118 @@ def ReceiveInfoFromMenu():
...
+    ReturnStatus = requests.post(
+        "http://localhost:26500/sendPrice/oid", json=OrderIDAndTotalPriceToFrontEnd)
 
-    status = requests.post("http://localhost:5000/sendPrice/oid", json=OrderIDAndTotalPriceToFrontEnd)
-
-    if status.status_code != 200:
+    if ReturnStatus.status_code != 200:
         return render_template("not200error.html")
 
     CoursesToThomas = json.dumps(contentjson)
```
[the remark this answers is not in the document]
Comment (TOsaland) [reply src/api.py]: Good catch

[the remark this answers is not in the document]
Comment (TOsaland) [reply src/api.py]: _Requested changes has been commited._
### **______**

**Pull request merged**